bodhi/server/models.py

```diff
@@ -1672,7 +1666,7 @@ class Update(Base):
     date_stable = Column(DateTime)
 
     # eg: FEDORA-EPEL-2009-12345
-    alias = Column(Unicode(32), unique=True, nullable=True)
+    alias = Column(Unicode(32), unique=True, nullable=False)
```

Comment:
With this change, we can mark commit as fixing #1714

Reply:
Fixed, and thanks!